scripts/tuple_tree_generator/metaschema.yml

```diff
@@ -108,6 +108,9 @@ definitions:
       const:
         type: boolean
         default: false
+      upcastable:
+        type: boolean
+        default: false
```

Comment:
I'm confused here: upcastability is the same as the type of the field having `abstract: true`.
Can you make an example showing how we use this?

Reply:
I just imitated what was done for containers without thinking too deeply: I just assumed there's a good reason to separate the two.

But I do agree that we should try dropping this `upcastable` flag everywhere if you can't think of a reason not to.

Reply:
Ah in the past `upcastable: true` was used in `SortedVector`s only. Now you need `UpcastablePointer<...>` elsewhere too.

OK, I see. Let's keep things as you're doing them.

We can put in backlog dropping `upcastable: true` since it could be automatically inferred by the fact that the inner type has `abstract: true`, but not now.